guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -40,33 +40,38 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
     var config = {
         restrict    : 'E',
         replace     : true,
+        scope: {
+            /**
+             * When isActive is set to true then the Clipboard data will be
+             * displayed in the Clipboard Editor. When false, the Clipboard Editor
+             * will not be updated with Clipboard data.
+             * 
+             * @type Boolean
+             */
+            isActive : '=?',
+        },
         templateUrl : 'app/clipboard/templates/guacClipboard.html'
     };
 
     // guacClipboard directive controller
     config.controller = ['$scope', '$injector', '$element',
             function guacClipboardController($scope, $injector, $element) {
 
-        /**
-         * The DOM element which will contain the clipboard contents within the
-         * user interface provided by this directive.
-         *
-         * @type Element
-         */
-        var element = $element[0];
-
+        // Check if isActive was defined. If not default to false
+        $scope.isActive = angular.isDefined($scope.isActive) ? $scope.isActive : false;
```

Comment:
What is this logic for? When would `$scope.isActive` be defined at controller initialization time?

Reply:
This was for when I made isActive an optional argument